int/api/websites/webcreator.go

```diff
@@ -202,53 +127,69 @@ func CreateUploadMissingChunksHandler(app *fyne.App) func(params operations.Webs
 
 //nolint:nolintlint,ireturn,lll
 func websiteUploadMissingChunksHandler(params operations.WebsiteUploadMissingChunksParams, app *fyne.App) middleware.Responder {
-	wallet, err := wallet.Load(params.Nickname)
+	wallet, _ := loadAndUnprotectWallet(params.Nickname, app)
+	archive, _ := readAndCheckArchive(params.Zipfile, app)
+
+	_, err := website.UploadMissedChunks(params.Address, archive, wallet, params.MissedChunks)
 	if err != nil {
-		return operations.NewWebsiteCreatorUploadInternalServerError().
-			WithPayload(
-				&models.Error{
-					Code:    errorCodeGetWallet,
-					Message: err.Error(),
-				})
+		return createInternalServerError(errorCodeWebCreatorUpload, err.Error())
+	}
+
+	return operations.NewWebsiteUploadMissingChunksOK().
+		WithPayload(&models.Websites{
+			Name:         "Name",
```

Comment:
This fell weird. Is it normal to set a static "Name" to Website.Name ?

Reply:
our struct Websites contains Name, Address and BrokenChunks, the three of it are not used everywhere, instead of "Name" i could use nil or create a new struct what do you think ?

Reply:
In that case you can:
- find the proper name of the siteweb if possible
- use an empty string
- change the type of this field to *string and return nil when the name is unknown